managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+
+### Adding Credentials
+
+Credentials are used by {{ site.data.product.title_short }} for any authentication that is required when running embedded workflows, and when importing workflows from a version control system.
+
+{{ site.data.product.title_short }} can store credentials that are used by embedded workflows. Credentials that are saved in {{ site.data.product.title_short }} are matched and executed with a workflow when it runs.
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Credentials**.
+
+2.  Click **Configuration**, then ![Add New Credential](../images/1862.png) (**Add New Credential**).
+
+3.  Provide a **Name** for the credential.
+
+4.  Select the **Credential Type**. More fields appear depending on the type chosen.
+
+5.  Click **Add**.
+
+To view the added credentials, return to the **Automation** > **Embedded Workflows** > **Credentials** page. The list of credentials is displayed. To view a summary of details for a credential, click the credential to open the *Summary* page for that credential.
+
+If you need to edit a credential, select the credential from the list of credentials. Then, click **Configuration** > **Edit this Credential**. After you complete your edits, click **Save**.
+
+If you need to delete a credential, select the credential from the list of credentials. Then, click **Configuration** > **Remove selected Credentials from Inventory**.
+
+#### Credential Types
+
+Each credential type that is used by {{ site.data.product.title_short }} for embedded workflows is detailed in the following sections.
+
+##### SCM
+
+SCM (source control Management) credentials are used with Projects to clone and update local source code Git repositories. Source Control credentials have several attributes that can be configured:
+
+- **Username**: The username to use in conjunction with the source control system.
+
+- **Password**: The password to use in conjunction with the source control system.
+
+- **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+
+- **Private Key**: Copy or drag-and-drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Authoring a workflow
+
+You can configure workflows for use in the following areas during service catalog creation:
+
+- **Dynamic dialog fields** - You (or your users) can attach workflows to a field so that the field becomes populated or refreshed with the results from the running of the workflow.
+- **Service Entry Points** - You (or your users) can set and use workflows for provisioning, reconfiguring, and retiring services.
+
+The Automate model provides flexibility to not only change parts of the provisioning process, but also to allow you to automate other operational tasks. You can create and use your own embedded workflows as needed to automate your operational tasks.
```

Comment:
2x `automate other operational tasks` feels repetitive repetitive

Reply:
I have rewritten this content to remove  the duplication: 
You can create and use embedded workflows as needed to not only change parts of the provisioning process, but also to automate other operational tasks.